styles/components/app-card.html

```diff
@@ -0,0 +1,17 @@
+<section class="app-card">
+  <button class="app-card__bookmark"></button>
+  <h4 class="app-card__title">Question</h4>
```

Comment:
because of semantic html i think here is h2 needed 😃

Reply:
Thanks for checking! I fixed the issue :)